File: energybalance_pkg/test/f90/test_diffusionlimitedevaporation.f90
!Test generation

PROGRAM test
    USE Diffusionlimitedevaporationmod
    REAL:: deficitOnTopLayers
    REAL:: soilDiffusionConstant
    REAL:: diffusionLimitedEvaporation
    print *, "--------test_test1_DiffusionLimitedEvaporation-------"
    soilDiffusionConstant = 4.2
    deficitOnTopLayers = 5341
    call model_diffusionlimitedevaporation(deficitOnTopLayers,  &
            soilDiffusionConstant, diffusionLimitedEvaporation)
    print *, diffusionLimitedEvaporation

END PROGRAM

File: energybalance_pkg/test/py/test_diffusionlimitedevaporation.py
'Test generation'

from Diffusionlimitedevaporation import *
from math import *
import numpy as np



def test_test1():
    params= model_diffusionlimitedevaporation(
    soilDiffusionConstant = 4.2,
    deficitOnTopLayers = 5341,
     )
    diffusionLimitedEvaporation_estimated = round(params, 3)
    diffusionLimitedEvaporation_computed = 6605.505
    assert (diffusionLimitedEvaporation_estimated == diffusionLimitedEvaporation_computed)